website/src/pages/index.js

```diff
@@ -50,6 +49,11 @@ function HomepageSubHeader() {
   );
 }
 
+export function Section({ element = 'section', children, className, background = 'light' }) {
```

Comment:
Unused code, probably.

Reply:
Yes, and I removed from this file and in the homepage branch, as well. Thanks.